shell/common/switches.cc

```diff
@@ -404,6 +406,16 @@ Settings SettingsFromCommandLine(const fml::CommandLine& command_line) {
       settings.dart_flags.push_back(flag);
     }
   }
+  if (std::find(settings.dart_flags.begin(), settings.dart_flags.end(),
```

Comment:
See the comment above about adding to shell switches instead of Dart switches.

Reply:
Or not - intentionally. They do work as is, but are not advertised as I don't know if we want them to become part of our public API unless we plan to support them beyond the initial testing and buy-in phase.